app/models/form/lettings/questions/managing_organisation.rb

```diff
@@ -12,10 +12,15 @@ def initialize(id, hsh, page)
 
   def answer_options
     opts = { "" => "Select an option" }
+
     return opts unless ActiveRecord::Base.connected?
     return opts unless current_user
     return opts unless log
 
+    if log.managing_organisation.present?
+      opts = opts.merge({ log.managing_organisation.id => log.managing_organisation.name })
+    end
+
```

Comment:
Whatever is currently in the database, we want to make sure is in the list of answer options.